resources/assets/js/videos/videoContainer.vue

```diff
@@ -676,9 +676,9 @@ export default {
                 Messages.danger('Error while loading video file.');
             }
         });
-        this.video.addEventListener('seeked', this.handleVideoSeeked);
+        this.video.addEventListener('timeupdate', this.handleVideoSeeked);
         this.video.addEventListener('pause', this.updateVideoUrlParams);
-        this.video.addEventListener('seeked', this.updateVideoUrlParams);
+        this.video.addEventListener('timeupdate', this.updateVideoUrlParams);
```

Comment:
This should still be seeked (because the URL is updated on pause or `seeked` but not while the video plays).